sys-apps/pcsc-lite/pcsc-lite-1.9.0.ebuild

```diff
@@ -28,7 +28,7 @@ REQUIRED_USE="^^ ( udev libusb ) ${PYTHON_REQUIRED_USE}"
 # No dependencies need the MULTILIB_DEPS because the libraries are actually
 # standalone, the deps are only needed for the daemon itself.
 DEPEND="libusb? ( virtual/libusb:1 )
-	udev? ( virtual/udev )
+	udev? ( virtual/libudev )
```

Comment:
This dependency needs `:=` operator, so either one of the following:
* `virtual/libudev:=`
* `virtual/libudev:0=`